backend/pkg/backend/backend.go

```diff
@@ -289,6 +290,12 @@ func Run() {
 
 func (b *Backend) handleHTTPTrace(ctx context.Context, trace *pluginsmodels.Telemetry, traceSource *models.TraceSource) error {
 	var err error
+	const defaultTraceSourceID = 0
```

Comment:
Perhaps move this on a more "common" place because it will be used from several place (TraceSampling also define it)